### PR TITLE
Add `monitoring` dbt project [DE-274]

### PR DESCRIPTION
**Pull request by jackwelty**
This PR migrates the one-off view in `/analysis/rpt_pipeline_health.sql` into a standalone dbt project for all data monitoring use cases. 

TODO (post-merge):
- [ ] manually deploy changes to `dbt_project.yml` and various `schema.yml` files

### CODE REVIEW

**Comment by emily-flambe** [comment]
oooooo gonna look now - real quick @jackwelty have you tried running this dbt locally?

**Comment by emily-flambe** [comment]
oh actually I just saw your other PR with the DAG! presumably that's running in dev if this PR is ready for review 🧐 👀

**Comment by jackwelty** [comment]
> oh actually I just saw your other PR with the DAG! presumably that's running in dev if this PR is ready for review 🧐 👀 

Yep green runs locally and in dev!

File: dbt-cta/monitoring/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# dbt Monitoring
+
+This models contained in this project do not pertain to any particulary sync, but
```
Comment (emily-flambe) [inline dbt-cta/monitoring/README.md:3]:
```suggestion
This models contained in this project do not pertain to any particular sync, but
```
lol

**Review by emily-flambe — COMMENTED** [review]
One suggestion, really an aesthetic thing - can we prefix the model folders with numbers indicating their ordering? so

`0_staging`
`1_intermediate`
`2_reporting`

Makes it slightly cognitively easier when looking at the models as a whole!

**Review by emily-flambe — DISMISSED** [review]
This is so thorough! I'm in shambles. Saw a typo 🫠 and suggested a change to folder names. Spiritually you have my approval to merge after committing those changes ✨

**Pull request merged**